server/resp/src/main/java/org/infinispan/server/resp/ByteBufferUtils.java

```diff
@@ -56,6 +59,48 @@ public static ByteBuf bytesToResult(byte[] result, ByteBufPool alloc) {
       return buffer;
    }
 
+   public static ByteBuf bytesToResult(List<byte[]> results, ByteBufPool alloc) {
```

Comment:
Reeeeally nitpicking here. Regarding the cast of the `Collection` to a `List` on `POP`, so feel free to ignore this one. 

If this and the other `bytesToResult` receive a `Collection<byte[]>` argument, the cast on the `POP` can be avoided, and a `Immutables.immutableListConvert` or similar could be applied instead.